hazelcast/src/main/java/com/hazelcast/monitor/impl/LocalWanPublisherStatsImpl.java

```diff
@@ -103,16 +105,24 @@ public void setSentCacheEventCounter(Map<String, DistributedObjectWanEventCounte
         this.sentCacheEventCounter = sentCacheEventCounter;
     }
 
-    public void setLastConsistencyCheckResults(
-            Map<String, ConsistencyCheckResult> lastMerkleTreeRootComparisonResults) {
-        this.lastConsistencyCheckResults = lastMerkleTreeRootComparisonResults;
+    public void setLastConsistencyCheckResults(Map<String, ConsistencyCheckResult> lastConsistencyCheckResults) {
+        this.lastConsistencyCheckResults = lastConsistencyCheckResults;
     }
 
     @Override
     public Map<String, ConsistencyCheckResult> getLastConsistencyCheckResults() {
         return lastConsistencyCheckResults;
     }
 
+    public void setLastSyncStats(Map<String, WanSyncStats> lastSyncStats) {
+        this.lastSyncStats = lastSyncStats;
+    }
+
+    @Override
+    public Map<String, WanSyncStats> getLastSyncStats() {
+        return lastSyncStats;
```

Comment:
Hm, we could probably add the check and sync results to the JSON but we can do that in a separate PR.